packages/components/src/types/components.ts

```diff
@@ -16,3 +16,8 @@ export type IsomerComponent = Static<typeof IsomerComponentsSchemas>
 export type IsomerComponentTypes =
   | keyof typeof IsomerComplexComponentsMap
   | "prose"
+
+export const LINK_TYPE_PAGE = "page"
+export const LINK_TYPE_EXTERNAL = "external"
+export const LINK_TYPE_FILE = "file"
+export const LINK_TYPE_EMAIL = "email"
```

Comment:
issue: this is a Studio-specific construct and should be added there. Components are flexible in the sense that it can take in any string (since they are forwarded verbatim to the `href` attribute), subject to the pattern defined for these 4 types of links.

Reply:
sounds good, thanks for pointing it out!